src/index.html

```diff
@@ -1,14 +1,70 @@
 <!doctype html>
-<html lang="en">
+<html lang = "en">
```

Comment:
```suggestion
<html lang="en">
```

Avoid using unnecessary spaces around the '=' in HTML attributes.